### PR TITLE
Comment feature

### PR DESCRIPTION
**Pull request by ash-11**
Comment feature with permanent storage.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package com.google.sps.data;
+
+public class Comment{
```
Comment (rahulnagurtha) [inline portfolio/src/main/java/com/google/sps/data/Comment.java:3]: Optional: If you had read about making every entity Immutable (from Effective Java), then Builder pattern makes sense here.

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/list-comments")
+public class ListCommentServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Query query = new Query("Comment");
+        
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```
Comment (rahulnagurtha) [inline portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java:43]: You can get rid of the intermediate variable "datastore".

**Review by rahulnagurtha — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java (hunk 2 above)
Comment (rahulnagurtha) [reply portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java:43]: Related article: https://techbeacon.com/app-dev-testing/why-unnecessary-variables-are-bad-your-code

File: portfolio/src/main/java/com/google/sps/data/Comment.java (hunk 1 above)
Comment (ash-11) [reply portfolio/src/main/java/com/google/sps/data/Comment.java:3]: Yes, Will do.

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java (hunk 2 above)
Comment (ash-11) [reply portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java:43]: Okay!

**Pull request merged**